hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_v2_excl.el

```diff
@@ -0,0 +1,32 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Manually exclude invalid state related FSM until V2S milestone is reached.
```

Comment:
Could you add a TODO here (which we'll find with grep!) to remind us to remove this file once the V2S tests are up and running?

Reply:
Good point, thanks for pointing that out.